components/class_loader.rst

```diff
@@ -10,6 +10,12 @@ The ClassLoader Component
 Usage
 -----
 
+.. caution::
+
+    If your application uses Composer's class loading mechanism and PHP 7 caching
+    features, there's no need to use this component to load classes. That's why
+    this component may be deprecated anytime soon.
```

Comment:
I've reworded it ... because the component is deprecated as of https://github.com/symfony/symfony/pull/21353